VolleyPaint/Assets/Scripts/Player/Shooting/PlayerShoot.cs

```diff
@@ -57,7 +58,7 @@ void Start()
     // Update is called once per frame
     void Update()
     {
-        if (!IsOwner) return;
+        //if (!IsOwner) return;
```

Comment:
uncomment this too and add toggle variable

Reply:
Got it